classes/ActionScheduler_ListTable.php

```diff
@@ -181,9 +181,9 @@ public function column_args( array $row ) {
 	 */
 	public function column_comments( array $row ) {
 		echo '<div id="log-' . $row['ID'] . '" class="log-modal hidden" style="max-width:800px">';
-		echo '<h3>' . sprintf( __( 'Log entries for %d', 'action-scheduler' ),  $row['ID'] ) . '</h3>';
+		echo '<h3>' . esc_html( sprintf( __( 'Log entries for %d', 'action-scheduler' ),  $row['ID'] ) ) . '</h3>';
 		foreach ( $row['comments'] as $log ) {
-			echo '<p>' . $log->get_message() . '</p>';
+			echo '<p><strong>' . esc_html( $log->get_date() ) . '</strong> ' . esc_html( $log->get_message() ) . '</p>';
```

Comment:
Ideally this should display the date's timezone to avoid ambiguity. If the suggestion to make `ActionScheduler_LogEntry::get_date()` return a `DateTime` is taken onboard, the code here can be responsible for displaying that.